apps/ledger-live-desktop/src/renderer/modals/Send/steps/StepRecipient.tsx

```diff
@@ -31,8 +34,14 @@ const StepRecipient = ({
   onChangeNFT,
   maybeNFTCollection,
 }: StepProps) => {
-  if (!status) return null;
-  const mainAccount = account ? getMainAccount(account, parentAccount) : null;
+  if (!status || !account) return null;
+  const mainAccount = getMainAccount(account, parentAccount);
+  // for ethereum family, check if there is a stuck transaction. If so, display a warning panel with "speed up or cancel" button
+  const [stuckAccount, stuckParentAccount, stuckOperation] = getEthStuckAccountAndOperation(
+    account,
+    parentAccount,
+  );
```

Comment:
If I'm correct with my proposal for `getEthStuckAccountAndOperation` this would have to be updated

Reply:
fixed. Please review again. Thanks